README.md

```diff
@@ -13,33 +13,33 @@ Role Variables
 
 Available variables are listed below, along with default values (see `defaults/main.yml`):
 
-## server
+## consul_server
 
 Install consul on server or not.
 
-Default is `false`.
+Default is `true`.
```

Comment:
Not really sure myself. But from what I've gather after looking back at `ansible-playbooks`, some hosts redefine the setting to true to provision it completely. I'm thinking other hosts that don't really need the service leave it as false but still uses some of the variables defined in the role.